dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry10/internal/rest/TelemetryServletFilter.java

```diff
@@ -101,14 +99,16 @@ private Instrumenter<ServletRequest, ServletResponse> getInstrumenter() {
     }
 
     private Instrumenter<ServletRequest, ServletResponse> createInstrumenter() {
+        // Check if the HTTP tracing should be disabled
+        boolean httpTracingDisabled = config.getOptionalValue("otel.trace.http.disabled", Boolean.class).orElse(false);
         OpenTelemetryInfo otelInfo = OpenTelemetryAccessor.getOpenTelemetryInfo();
         if (tc.isDebugEnabled()) {
             Tr.debug(tc, "otelInfo.getEnabled()=" + otelInfo.getEnabled());
         }
```

Comment:
Please trace the value of `httpTracingDisabled`

Reply:
Done